main.py

```diff
@@ -343,12 +357,14 @@ def export_assets(request):
     Returns:
         A tuple containing a success message and HTTP status code.
     """
-    logging.info("Received export assets request")
+    setup_logging()
```

Comment:
does this need to get called with publish? you can put this in global. also you are returning a logging_client and not using it. i assume monkey patch magic?

Reply:
correct need it non-global so I can patch it in the tests otherwise the test try to use GCP logging which requires creds etc